flexget/plugin.py

```diff
@@ -108,7 +108,7 @@ def __str__(self):
 
 
 # TODO: move to utils or somewhere more appropriate
-class internet:
+class Internet:
```

Comment:
Don't think I like this. Whether it's implemented as a class or function seems to be an implementation detail to me. It acts like a function in usage.

Reply:
Most classes should have capitalized first letters. Should I use `noqa` here or disable the rule?

Reply:
I think noqa is okay for this one and `cache`